src/osd/ECTransaction.cc

```diff
@@ -241,7 +241,12 @@ ECTransaction::WritePlanObj::WritePlanObj(
        */
     }
 
-    if (!reads.empty()) {
+    /* If the read is not empty AND the object is not going tbe deleted as
```

Comment:
The first sentence in the commit description seems wrong
(commit: osd: Multiple fixes to optimized EC and peering)

Reply:
Sorry - I used the wrong title. My problem is with this sentence:
"...both write an entire object and write them."